lib/js/components/SelectList/SelectList.stories.ts

```diff
@@ -2,16 +2,18 @@ import SelectList from './SelectList.vue';
 import { SELECT_LIST_SIZES } from './SelectList.consts';
 import { ICONS } from '../Icon';
 
-import { Args, ArgTypes, Meta, StoryFn } from '@storybook/vue';
+import { Args, ArgTypes, Meta, StoryFn } from '@storybook/vue3';
 
 export default {
 	title: 'Components/SelectList',
 	component: SelectList,
 } as Meta<typeof SelectList>;
 
-const StoryTemplate: StoryFn<typeof SelectList> = (argTypes) => ({
+const StoryTemplate: StoryFn<typeof SelectList> = (args) => ({
 	components: { SelectList },
-	props: Object.keys(argTypes),
+	setup() {
+		return { ...args };
+	},
 	template: '<select-list v-model="value" :items="items" :size="size"></select-list>',
 	data() {
```

Comment:
Something is wrong here:

![Screenshot 2023-01-27 at 16 08 49](https://user-images.githubusercontent.com/7030884/215119520-d73ea642-4ac1-445e-adba-1a368794cc61.png)

Reply:
There is a lot of warnings about it in test logs, too.

Reply:
`icon` prop passed to https://github.com/bethinkpl/design-system/blob/0f5267f6737eb1a363656c4cedaed9a2d386c75a/lib/js/components/Icons/Icon/Icon.vue#L36-L38 is of a `Proxy` type, so that's probably the reason it doesn't validate properly.
![Screenshot 2023-02-10 at 17 07 07](https://user-images.githubusercontent.com/7030884/218139268-248a5bf5-03ab-42fc-bba2-c5fc6bfb54f8.png)